package.json

```diff
@@ -26,30 +26,31 @@
     "express": "^4.10.7",
     "glob": "^4.3.2",
     "http-proxy-middleware": "~0.11.0",
+    "ip": "~1.1.2",
```

Comment:
this was added by Joel, see commit:  https://github.com/RackHD/on-http/commit/138f6f265d836020af4020feb70f3dab83efea3a, I just put it in alphabetical order.